cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -27,11 +28,14 @@ public void setUp() throws Exception {
         contentfulCmsService = ContentfulCmsService.of(SPACE_ID, TOKEN);
     }
 
-    @Test(expected=ExecutionException.class)
-    public void whenCouldNotFetchEntry_thenReturnOptionalEmpty() throws Exception {
+    @Test
+    public void whenCouldNotFetchEntry_thenThrowException() throws Exception {
         final ContentfulCmsService cmsService = ContentfulCmsService.of("", "");
         final CmsIdentifier cmsIdentifier = CmsIdentifier.of("entryType:entryKey.fieldName");
-        waitAndGet(cmsService.get(SUPPORTED_LOCALES, cmsIdentifier));
+
+        Throwable thrown = catchThrowable(() -> waitAndGet(cmsService.get(SUPPORTED_LOCALES, cmsIdentifier)));
+
+        assertThat(thrown).isInstanceOf(ExecutionException.class).hasMessageContaining("Unauthorized");
```

Comment:
Could you create a ticket for it? So I will remember to do this in short future. 
(that's the purpose of creating ticket's, right?)